src/App.js

```diff
@@ -158,6 +159,7 @@ function App() {
           <Route component = {Mission} exact path = '/mission'/>
```

Comment:
This component makes sense as the MissionsScreen. We should probably make more of a distinction between Mission and MissionsScreen.